pkgs/development/compilers/extempore/default.nix

```diff
@@ -0,0 +1,114 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchurl,
+  cmake,
+  pkg-config,
+  pcre,
+  libuv,
+  portaudio,
+  boost,
+  zlib,
+  python3,
+  libffi,
+  xorg,
+  mesa,
+  libGL,
+}:
+
+let
+  llvm-patched = stdenv.mkDerivation {
```

Comment:
@NixOS/llvm Is this a good way to add new packages needing old llvm?

Reply:
No, anything under LLVM 12 isn't supported at all. v3.8.0 hasn't even been supported by upstream in many years.